lib/utils/initialAppConfig.ts

```diff
@@ -160,7 +160,7 @@ export function generateAppConfig (mlspaceConfig: MLSpaceConfig) {
                 'applications': {'L': applicationList}
             }
             },
-            'DisabledInstanceTypes': {'M': {
+            'EnabledInstanceTypes': {'M': {
```

Comment:
It will only impact current devs, it will require a manual update to the app config in our database so this field is changed to `EnabledInstanceTypes`. 

We also have a backlog task to handle migrations in the future so that once we release this to customers we can handle this gracefully without manual intervention needed from a customer.